Makefile

```diff
@@ -216,6 +230,36 @@ lint: ## Lint Rust code
 test: ## Run tests
 	cargo test -vv
 
+.PHONY: test.gencert
+test.gencert: cfssl cfssljson
+	$(CFSSL) gencert \
+		-initca $(TEST_CERTS_PATH)/ca-csr.json | $(CFSSLJSON) -bare ca
+	$(CFSSL) gencert \
+		-ca=ca.pem \
+		-ca-key=ca-key.pem \
+		-config=$(TEST_CERTS_PATH)/ca-config.json \
+		-profile=server \
+		$(TEST_CERTS_PATH)/server-csr.json | $(CFSSLJSON) -bare server
+	$(CFSSL) gencert \
+		-ca=ca.pem \
+		-ca-key=ca-key.pem \
+		-config=$(TEST_CERTS_PATH)/ca-config.json \
+		-profile=clinet \
+		$(TEST_CERTS_PATH)/client-csr.json | $(CFSSLJSON) -bare client
+	mv *.pem *.csr $(TEST_CERTS_PATH)
+
+.PHONY: test.rmcert
+test.rmcert:
+	rm $(TEST_CERTS_PATH)/{*.pem,*.csr}
+
+.PHONY: test.dataplane.integration
+test.dataplane.integration:
+	go clean -testcache
+	TEST_CERTS_PATH=./$(TEST_CERTS_PATH) \
+	BLIXT_DATAPLANE_IMAGE=$(BLIXT_DATAPLANE_IMAGE):$(TAG) \
+	GOFLAGS="-tags=dataplane_tests" go test -race -v ./test/integration/...
+	$(MAKE) test.rmcert
```

Comment:
I am happy mpt to remove them since this test will mostly run in ephemeral CI environments. But I think we _should keep_ the targets for creating them. We want to have an automated way to create self-signed certs.

Reply:
Right, exactly what I was getting at. Keep the _targets_ but don't do cleanup automatically here. :+1: